app/src/components/market/common/market_data/index.tsx

```diff
@@ -89,26 +95,46 @@ export const MarketData: React.FC<Props> = props => {
       ? formatBigNumber(runningDailyVolumeByHour[Math.floor(Date.now() / (1000 * 60 * 60)) % 24], currency.decimals)
       : '0'
 
+  const totalVolume = formatBigNumber(collateralVolume, currency.decimals)
+
   return (
     <MarketDataWrapper>
       <MarketDataItem>
         <MarketDataItemTop>
-          {moment(resolutionTimestamp).format('DD.MM.YYYY - H:mm zz')} {timezoneAbbr}
+          {liquidity} {currency.symbol}
```

Comment:
`liquidity` should be wrapped in `formatNumber()` so it's displayed as: `50,010.00` as opposed to the current `50010.00`.